tests/vmi_lifecycle_test.go

```diff
@@ -1598,7 +1598,7 @@ func shouldUseEmulation(virtClient kubecli.KubevirtClient) bool {
 }
 
 func renderPkillAllJob(processName string) *k8sv1.Pod {
-	return tests.RenderJob("vmi-killer", []string{"pkill"}, []string{"-9", processName})
+	return tests.RenderPod("vmi-killer", []string{"pkill"}, []string{"-9", processName})
```

Comment:
Same here, the function name says `Job` but in it uses `Pod`.

Reply:
We missed this one or it came back?

Reply:
or rebase...